### PR TITLE
[Tests] Minor fixtures cleanup

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

I have added quotation to the strings and remove unnecessary object config

### CODE REVIEW

File: tests/DataFixtures/ORM/resources/checkout.yml [hunk 1]
```diff
@@ -50,11 +50,11 @@ Sylius\Component\Core\Model\ProductVariant:
 
 Sylius\Component\Product\Model\ProductVariantTranslation:
     sw_mug_translation:
-        locale: en_US
+        locale: "en_US"
         name: 'Star wars mug'
```
Comment (Zales0123) [inline tests/DataFixtures/ORM/resources/checkout.yml]: Maybe you could also change all apostrophes to quotation marks?

**Comment by pamil** [comment]
There's a typo probably, Travis failing :)

**Comment by pamil** [comment]
@lchrusciel are you willing to rebase this one?

**Comment by lchrusciel** [comment]
@pamil done.

**Comment by Zales0123** [comment]
@lchrusciel something is still wrong here :/ with test fixtures I suppose

File: tests/DataFixtures/ORM/resources/checkout.yml [hunk 2]
```diff
@@ -70,65 +70,68 @@ Sylius\Component\Core\Model\ShippingMethod:
...
+        channel: ["@channel"]
+        currentLocale: "en_US"
+        currentTranslation: "@dhl_translation"
+        channels: ["@channel"]
 
 Sylius\Component\Shipping\Model\ShippingMethodTranslation:
-    upsTranslation1:
-        name: 'UPS'
-        locale: 'en_US'
+    ups_translation:
```
Comment (Zales0123) [inline tests/DataFixtures/ORM/resources/checkout.yml]: Yes, you've changed it here but not in phpunit tests 🐃

**Comment by Zales0123** [comment]
Thank you Łukasz!

**Pull request merged**